packages/FormElement/README.md

```diff
@@ -28,7 +43,22 @@ import DatePicker from "@paprika/date-picker";
 import FormElement from "@paprika/form-element";
 
 <FormElement label="Name">
-  <DatePicker onChange={changeHandler} />
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <DatePicker
+        id={idForLabel}
+        onChange={handleChange}
+        value={value}
+        placeholder="Form placeholder"
+        aria-describedby={ariaDescribedBy}
+        aria-required={hasRequiredLabel}
+        hasError={false}
+        isDisabled={isDisabled}
+        isReadOnly={isReadOnly}
+        size={size}
```

Comment:
This isn't quite right, according to the [README](https://github.com/acl-services/paprika/blob/master/packages/DatePicker/README.md). But, this doesn't need to document that component, so I'd just stick to the simplest case (still might need to show how to propagate `ariaDescribedBy` to the `<DatePicker.Input>` though?)